src/Render/Program.cs

```diff
@@ -46,10 +47,13 @@ internal static class Program {
 			var text = await File.ReadAllTextAsync( filename );
 
 			var doc = MarkdownFactory.Parse( text );
-
 			doc.ApplyD2LTweaks();
-
-			MarkdownFactory.Render( outputHtml, doc );
+			var html = MarkdownFactory.RenderToString( doc );
+			var renderer = new TemplateRenderer( 
+				await File.ReadAllTextAsync( "Templates/page.html" )
+			);
+			var formatted = await renderer.RenderAsync( "TODO: Get Title", html );

```

Comment:
```suggestion
			var formatted = await renderer.RenderAsync(
				title: "TODO: Get Title",
				content: html
			);

```

Named parameters are ideal when there are multiple arguments of the same type, or literal values. :)

Reply:
Relatedly, I opened #21